src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -29,7 +29,21 @@
   {{ form_start(currencyForm, {
     'attr': {
       'id': 'currency_form',
-      'data-get-cldr-data': path('admin_currencies_get_reference_data', {'currencyIsoCode': 'CURRENCY_ISO_CODE'}),
+      'data-reference-url': path('admin_currencies_get_reference_data', {'currencyIsoCode': 'CURRENCY_ISO_CODE'})|replace({'/CURRENCY_ISO_CODE': '{/id}'}),
+      'data-languages': languages|json_encode|replace("'", "&#39;")|raw,
+      'data-translations': {
+        '1. Enter symbol': '1. Enter symbol'|trans({}, 'Admin.International.Feature'),
+        '2. Choose format': '2. Choose format'|trans({}, 'Admin.International.Feature'),
+        'Customize symbol and format': 'Customize symbol and format'|trans({}, 'Admin.International.Feature'),
+        'Apply': 'Apply'|trans({}, 'Admin.Actions'),
+        'Cancel': 'Cancel'|trans({}, 'Admin.Actions'),
+        'Close': 'Close'|trans({}, 'Admin.Actions'),
+        'Language': 'Language'|trans({}, 'Admin.International.Feature'),
+        'Example': 'Example'|trans({}, 'Admin.International.Feature'),
+        'Edit symbol / format': 'Edit symbol / format'|trans({}, 'Admin.International.Feature'),
+        'Reset settings': 'Reset settings'|trans({}, 'Admin.International.Feature'),
+        'Price: %price% (Special discount %discount%)': 'Price: %price% (Special discount %discount%)'|trans({}, 'Admin.International.Feature')
```

Comment:
🤔

Reply:
What's the problem? Is it the placeholders? Maybe use some `[1]` instead? I'm not sure about the usual convetion but I thought I had already some `%placeholder%` sometimes

Reply:
Sorry, I should have explained: I don't understand why this price and discount wording appears at this point. It doesn't seem related to the currency block to me, does it?